src/styles/blocks/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  margin: 0 auto;
```

Comment:
```suggestion
```
Don't use outer spacings (margins) for BEM blocks. `.container` is a block here

Reply:
In video, Misha said that we can use this container with margins in this case, but we should remember that we can`t use this block like BEM-block.  We use it to set max-width for all blocks and to set them in the middle

Reply:
Ok